crates/store/re_log_types/src/path/entity_path_filter.rs

```diff
@@ -309,15 +295,13 @@ impl EntityPathFilter {
     /// Each line is a rule.
     ///
     /// The first character should be `+` or `-`. If missing, `+` is assumed.
-    /// The rest of the line is trimmed and treated as an entity path.
+    /// The rest of the line is trimmed and treated as an entity path after variable substitution through [`Self::resolve_forgiving`]/[`Self::resolve_strict`].
     ///
     /// Conflicting rules are resolved by the last rule.
-    pub fn parse_strict(
-        rules: &str,
-        subst_env: &EntityPathSubs,
-    ) -> Result<Self, EntityPathFilterParseError> {
-        let split_rules = split_whitespace_smart(rules);
-        Self::from_query_expressions_strict(split_rules, subst_env)
+    #[allow(clippy::unnecessary_wraps)]
+    pub fn parse_strict(rules: &str) -> Result<Self, EntityPathFilterError> {
+        // TODO(andreas): Do some error checking here.
```

Comment:
Did you mean that for this PR? for later?

Reply:
later